src/resources/postcss/components/full/_search.pcss

```diff
@@ -96,6 +107,22 @@
 		}
 	}
 
+	.tribe-events-c-search__button {
+		background-color: var(--tec-color-background-events-bar-submit-button);
+		color: var(--tec-color-text-events-bar-submit-button);
+
+		&:active {
+			background-color: var(--tec-color-background-events-bar-submit-button-active);
+			color: var(--tec-color-text-events-bar-submit-button-active);
+		}
```

Comment:
have this come after hover and focus, if this is before those two, it won't take effect.